Examples/Tests/PEC/analysis_pec_mr.py

```diff
@@ -0,0 +1,84 @@
+#! /usr/bin/env python
+
+#
+#
+# This file is part of WarpX.
+#
+# License: BSD-3-Clause-LBNL
+
+
+# This is a script that analyses the simulation results from
+# the script `inputs.3d`. This simulates a sinusoindal wave.
```

Comment:
Do you mean `Examples/Tests/PEC/inputs_field_PEC_mr_3d`?  Also if this is mesh refinement is the comment below accurate?

Reply:
Not sure how the comment below would be inaccurate due to MR.